### ISSUE TITLE
[bug] wp-shortcodes not enquing option types css/js

### ISSUE THREAD

**Issue by danyj** [ThemeFuse/Unyson#2544]
@andreiglingeanu  

If  option type is not used anywhere in page/post settings

the option type `_enqueue_static  `does not run, 

in  taxonomy/category .php

https://github.com/ThemeFuse/Scratch-Theme/blob/master/scratch-parent/framework-customizations/theme/options/taxonomies/category.php

add **ONLY** editor with shortcodes  set to true 

```
			'test' => array(
				'type' => 'wp-editor',
				'size' => 'large',
				'editor_height' => 100,
				'editor_type' => 'tinymce',
				'wpautop' => true,
				'shortcodes' => true,
				'value' => 'This is dummy heading text',
				'label' => __('Heading', 'fw'),
				'desc' => esc_html__('Add heading text', 'fw')
			),
```

in editor insert  any shortcode that has your custom option type,  or addable popup ( or other special option type )   and once added open that shortcode. 

Shortcode  options CSS/JS will not load , unless you use the  option type somewhere in category.php 

even some of Unyson default options like **addable popup** do not load their scripts

![screenshot_14](https://cloud.githubusercontent.com/assets/2340172/25316585/177ccd32-286a-11e7-829a-c066bf283720.jpg)

**Comment by andreiglingeanu** [comment]
Yeah, we encountered that too. That's a known bug and there's not a proper fix for that yet. We can only add sub-optimal hack for that.

I'll think about it.

cc. @GheorgheP

**Comment by danyj** [comment]
@ViorelEremia  waiting for fix please respond
